src/style.css

```diff
@@ -1 +1,47 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src:
+    url(fonts/Avenir-Book.ttf) format(ttf),
+    url(fonts/Avenir-Heavy.ttf) format(ttf),
+    url(fonts/Avenir.ttc) format(ttc);
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px #3d4e611a;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  border: solid 1px #e1e7ed;
+  border-radius: 3px;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-bar__big {
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here, the padding is set on both top and bottom which may lead to unexpected layout behavior. Consider using padding only on one side, or ensure that this is the desired effect.